cristin-project/src/main/java/no/unit/nva/cristin/projects/query/organization/QueryCristinOrganizationProjectHandler.java

```diff
@@ -100,12 +102,13 @@ protected SearchResponse<NvaProject> processInput(Void input, RequestInfo reques
         return cristinApiClient.listOrganizationProjects(requestQueryParameters);
     }
 
-    private Map<String, String> extractQueryParameters(RequestInfo requestInfo) throws BadRequestException {
-        var requestQueryParameters = Map.of(
+    private ConcurrentHashMap<String, String> extractQueryParameters(RequestInfo requestInfo)
```

Comment:
I think you can keep the abstract type Map as it is, and rather only use the implementation when creating instance of it.

Example: `Map<String, String> map = new ConcurrentHashMap<>()`
Another: 
```
private Map<String, String> extractQueryParameters() {
  return new ConcurrentHashMap<>(with the data)
}  
```